homeassistant/components/xbox/__init__.py

```diff
@@ -55,10 +62,10 @@ async def async_migrate_unique_id(hass: HomeAssistant, entry: XboxConfigEntry) -
         # change generic entry name to user's gamertag
         return hass.config_entries.async_update_entry(
             entry,
-            unique_id=entry.runtime_data.client.xuid,
+            unique_id=entry.runtime_data.status.client.xuid,
             title=(
-                entry.runtime_data.data.presence[
-                    entry.runtime_data.client.xuid
+                entry.runtime_data.status.data.presence[
+                    entry.runtime_data.status.client.xuid
```

Comment:
Can a variable that stores `entry.runtime_data.status` be created and used in the method call instead of accessing multiple times to `entry.runtime_data.status`?